XIV on Mac/Util.swift

```diff
@@ -159,6 +159,7 @@ struct Util {
         env["DXVK_FRAME_RATE"] = dxvkOptions.getMaxFramerate()
         env["XL_WINEONLINUX"] = "true"
         env["XL_WINEONMAC"] = "true"
+        env["MVK_CONFIG_FULL_IMAGE_VIEW_SWIZZLE"] = "0"
```

Comment:
This forcefully disables texture swizzling which prevents the game from launching on Intel GPUs.